DQMServices/FileIO/scripts/compareDQMOutput.py

```diff
@@ -89,7 +89,7 @@ def upload(files):
             print('Exception uploading a file: %s' % ex)
 
 def generate_summary_html(output_dir, pr_list, summary_dir):
-    template_file = open(os.path.join(os.path.dirname(__file__), 'dqm-histo-comparison-summary-template.html'), 'r')
+    template_file = open(os.path.join(os.getenv('CMSSW_BASE'), 'src', 'DQMServices', 'FileIO', 'scripts', 'dqm-histo-comparison-summary-template.html'), 'r')
```

Comment:
@andrius-k , please note that DQMServices/FileIO package is not always checked out (e.g example for PR tests we always pick it up from release area) so the change is not going to work. So please first check if it exists under CMSSW_BASE otherwise read it from CMSSW_RELEASE_BASE

Reply:
@smuzaffar fixed it, thanks!